.pre-commit-config.yaml

```diff
@@ -12,34 +12,34 @@ repos:
         language: system
         types: [rust]
         args: ["--", "--check"]
-      # - id: cargo-check
-      #   name: cargo check
-      #   description: Check the package for errors.
-      #   entry: cargo check
-      #   language: system
-      #   types: [rust]
-      #   pass_filenames: false
-      # - id: clippy
-      #   name: clippy
-      #   description: Lint via clippy
-      #   entry: cargo clippy
-      #   language: system
-      #   args: ["--", "-D", "warnings"]
-      #   types: [rust]
-      #   pass_filenames: false
+      - id: cargo-check
+        name: cargo check
+        description: Check the package for errors.
+        entry: cargo check
+        language: system
+        types: [rust]
+        pass_filenames: false
+      - id: clippy
+        name: clippy
+        description: Lint via clippy
+        entry: cargo clippy
+        language: system
+        args: ["--", "-D", "warnings"]
+        types: [rust]
+        pass_filenames: false
```

Comment:
haha yeah I guess I can just run `--no-verify`. It is nice to have the rest of them run while in WIP, though.

Reply:
yep, but we'll get these working normally too (soon)

Reply:
Just an FYI, my branches will probably have a bunch of `--no-verify` while they're WIP. I tend to push before e.g. a string of meetings, and a lot of those states won't compile.

I think Quinn has also disabled a bunch of these checks for the same reason.

Reply:
Would it make sense to push some of these to a GH Action?

Reply:
I look at clippy as another extension of the type system :).